docs/_docs/installation/macos.md

```diff
@@ -3,156 +3,84 @@ title: Jekyll on macOS
-brew install rbenv
-
-# Set up rbenv integration with your shell
-rbenv init
-
-# Check your installation
-curl -fsSL https://github.com/rbenv/rbenv-installer/raw/main/bin/rbenv-doctor | bash
+brew install chruby ruby-install
 ```
 
-Restart your terminal to apply your changes.
-Next, you can install the Ruby version you want. Let's install the latest stable version:
+Install the latest stable version of Ruby:
 
 ```sh
-rbenv install {{ site.data.ruby.current_version }}
-rbenv global {{ site.data.ruby.current_version }}
-ruby -v
-{{ site.data.ruby.current_version_output }}
+ruby-install ruby
 ```
 
-That's it! Head over to [rbenv command references](https://github.com/rbenv/rbenv#command-reference) to learn how to use different versions of Ruby in your projects.
-
-## Install Jekyll
-
-After installing Ruby, install Jekyll and Bundler.
-
-### Local Install
-
-Install the bundler and jekyll gems:
+This will take a few minutes, and once it's done, configure your shell to automatically use `chruby`:
 
 ```sh
-gem install --user-install bundler jekyll
+echo "source /usr/local/opt/chruby/share/chruby/chruby.sh" >> ~/.zshrc
+echo "source /usr/local/opt/chruby/share/chruby/auto.sh" >> ~/.zshrc
+echo "chruby ruby-{{ site.data.ruby.current_version }}" >> ~/.zshrc
 ```
 
-Get your Ruby version:
+If you're on an M1 Mac, replace `/usr/local` with `/opt/homebrew`.
+If you're using Bash, replace `.zshrc` with `.bash_profile`. If you're not sure, read this external guide to [find out which shell you're using](https://www.moncefbelyamani.com/which-shell-am-i-using-how-can-i-switch/).
+
+Quit and relaunch Terminal, then check that everything is working:
 
 ```sh
 ruby -v
-{{ site.data.ruby.current_version_output }}
 ```
 
-Append your path file with the following, replacing the `X.X` with the first two digits of your Ruby version:
-
-```bash
-# If you're using Zsh
-echo 'export PATH="$HOME/.gem/ruby/X.X.0/bin:$PATH"' >> ~/.zshrc
+It should show {{ site.data.ruby.current_version_output }} or a newer version.
 
-# If you're using Bash
-echo 'export PATH="$HOME/.gem/ruby/X.X.0/bin:$PATH"' >> ~/.bash_profile
+Next, read that same external guide for important notes about [setting and switching between Ruby versions with chruby](https://www.moncefbelyamani.com/how-to-install-xcode-homebrew-git-rvm-ruby-on-mac/#how-to-install-different-versions-of-ruby-and-switch-between-them).
 
-# Unsure which shell you are using? Type
-echo $SHELL
-```
+## Install Jekyll
 
-Check that `GEM PATHS:` points to your home directory:
+After installing Ruby with chruby, install Jekyll and Bundler:
 
 ```sh
-gem env
+gem install bundler jekyll
 ```
 
-{: .note .info}
-Every time you update Ruby to a version in which the first two digits change, update your path to match.
-
-### Global Install
-
-{: .note .warning}
-We recommend not installing Ruby gems globally to avoid file permissions problems and using `sudo`.
-
-#### On Mojave (10.14)
+### Note about Jekyll and Ruby 3.x
```

Comment:
Let us drop this section entirely.
The docs already have this covered in a different page. No need to reinstate here, mostly because `webrick` gem will be installed automatically as a Jekyll dependency from the next minor version release onwards.